Initialized camera...
Failed to initialize motor system, trying again
System initialized... commencing tracking program
Traceback (most recent call last):
  File "C:\Users\Babai\Documents\python\MotionDetection\MotionDetection&Tracking.py", line 153, in <module>
    motion_detector()
  File "C:\Users\Babai\Documents\python\MotionDetection\MotionDetection&Tracking.py", line 134, in motion_detector
    cv.DrawContours(img, largest_contour, (0,255,0), (0,0,255), 1, 3)
UnboundLocalError: local variable 'largest_contour' referenced before assignment
